### ISSUE TITLE
Motion transition doesn't include crop

### ISSUE THREAD

**Issue by LukasW1337** [CatxFish/motion-filter#16]
Hey! 
When using the Motion transition with objects that are cropped, they just change size "suddenly" - and not smooth like the showed animation. I'm sure that this is a bug, and the showed animation was with objects changing size etc.

**Comment by CatxFish** [comment]
Yes , rotation ,bounding box, and cropped is not considered in current version . It will be fixed someday , but not recently. Right now the workaround way is not to use these feature if you want to use motion transition.

**Comment by LukasW1337** [comment]
Would it be possible to get these features soon? Maybe pay for them? :P

**Comment by CatxFish** [comment]
I'll make a pre-release after january 2.
